<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Jcal</class>
 <widget class="QMainWindow" name="Jcal">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900000</width>
    <height>900000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Jcal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #404040;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">centralwidget {
	background-color: #404040;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="AC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #DF0000;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #B22222;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_parenthesis">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closed_parenthesis">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divide">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiply">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #808080;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #707070;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equal">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>441</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: #606060;
	border-style: solid;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>1999999999</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="power">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>X^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="squred">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>X^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cube">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>X^3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="abs">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string>QPushButton {
	background-color: #FF8C00;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: #FF4500;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>abs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">menubar {
	background-color: #404040;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">statusbar {
	background-color: #404040;
	border-style: solid;
	border-width: 5px;
	border-radius: 10px;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
